config/v1.0/calico.yaml

```diff
@@ -114,13 +114,7 @@ spec:
             path: /var/run/calico
       tolerations:
         # Make sure calico/node gets scheduled on all nodes.
-        - effect: NoSchedule
```

Comment:
@caseydavenport I don't think this is needed for calico policy

Reply:
If the aws CNI is going to tolerate everything, shouldn't the calico policy do the same?